platform/qt/qt.cmake

```diff
@@ -7,21 +7,27 @@ set(MBGL_QT_VERSION_COMPATIBILITY 2.0.0)
 message(STATUS "Build type: ${CMAKE_BUILD_TYPE}")
 message(STATUS "Version ${MBGL_QT_VERSION}")
 
-option(MBGL_QT_STATIC "Build Mapbox GL Qt bindings staticly" OFF)
 option(MBGL_QT_LIBRARY_ONLY "Build only libraries" OFF)
+option(MBGL_QT_INSIDE_QT_PLUGIN "Build qmapboxgl as OBJECT library, so it can be bundled into separate single plugin lib." OFF)
```

Comment:
I guess we may still want to support Qt 5 for some time where object libraries would not work.

Reply:
Why object libraries would not work for Qt5?

Reply:
I thought object libraries only work with CMake. If this also works with qmake, then it's fine.

Reply:
MapLibre is mostly a drop-in replacement for MapBox (see https://github.com/ntadej/qt-geoservices-maplibre). If we use the old qmake-based build infrastruture for Qt 5 plugins, we still need to be able to link to some sort of static maplibre-gl-native library or libraries.

Reply:
A, ok, I see. But it should work because OBJECT library is only internal to the CMake build tree. Final lib will always be either a shared or static `qmapboxgl`.